CoreUtilities/src/au/gov/asd/tac/constellation/utilities/tooltip/TooltipUtilities.java

```diff
@@ -129,4 +136,70 @@ private static void selectActiveArea(TextInputControl control, List<TooltipProvi
             control.selectRange(s, e);
         }
     }
+
+    public static void activateTextInputControl(final InlineCssTextArea textArea, final TooltipPane tooltipPane) {
+
+        final int[] characterIndex = new int[1];
+        final TooltipNode[] tooltipNode = new TooltipNode[1];
+
+        // PLACEHOLDER CODE TO IMITATE TOOLTIPPANE POPUP. REMOVE LATER ONCE TOOLTIPPANE CONFIRMED WORKING.
+        Popup popup = new Popup();
+        Label popupMsg = new Label();
```

Comment:
Acknowledge this is placeholder code if still required these can be final (else the code block can just be omitted)